torch/distributed/device_mesh.py

```diff
@@ -396,6 +396,9 @@ class DeviceMesh:
             device_type (str): The device type of the mesh. Currently supports: "cpu", "cuda/cuda-like".
             mesh (ndarray): A multi-dimensional array or an integer tensor describing the layout
                 of devices, where the IDs are global IDs of the default process group.
+            _rank (int): (experimental/internal)
+                The global rank of the current process. If not provided, it will
+                be inferred from the default process group.
```

Comment:
Ohh, I totally misunderstood what this argument is for. 
I thought it was intended to describe "the rank of this process in the DeviceMesh to be created."
Like, if I'd like to create a DeviceMesh covering global ranks [100, 200), then rank 100 would set `_rank=0`, rank 101 would set `_rank=1`, etc.